app/js/main.js

```diff
@@ -93,7 +93,11 @@ var width  = 640,
     mkdirp = require("./lib/mkdirp"),
 
     // Sidebar
-    btnDirectoryChange = document.querySelector("#sidebar #btn-dir-change");
+    btnDirectoryChange = document.querySelector("#sidebar #btn-dir-change"),
+
+    // Shortcuts
+    controlKey = null,
```

Comment:
Remove `controlKey` from the global scope. It is only used in `loadMenu()` and should remain in that scope only.